torch/distributed/_shard/replicated_tensor.py

```diff
@@ -101,7 +112,8 @@ def dispatch_arg(arg):
             rs = func(*args, **kwargs)
             if func in get_default_nowrap_functions():
                 return rs
-            if all_replicated and isinstance(rs, torch.Tensor) and not isinstance(rs, cls):
+            if (all_replicated and isinstance(rs, torch.Tensor) and not isinstance(rs, cls)) or \
```

Comment:
N00b question, is the 'cls' here just `ReplicatedTensor`?

Reply:
Yes, let me update it so it is more clear.